include/torch-mlir/Dialect/Torch/IR/TorchOps.h

```diff
@@ -45,6 +45,21 @@ struct torch_constant_int_op_binder {
     return false;
   }
 };
+
+struct torch_constant_float_op_binder {
+  APFloat *bind_value;
```

Comment:
Prefer to use the primitive type `double` so that user of the API can use the result number directly for calculation. Like in `AtenEqFloatOp::fold`, `lhs.compare(rhs) == APFloat::cmpEqual` can be replaced with `lhs == rhs`.